mlflow/store/artifact/optimized_s3_artifact_repo.py

```diff
@@ -53,8 +53,9 @@ def __init__(
         addressing_style=None,
         s3_endpoint_url=None,
         s3_upload_extra_args=None,
+        tracking_uri=None,
     ):
-        super().__init__(artifact_uri)
+        super().__init__(artifact_uri, tracking_uri)
```

Comment:
is this correct? should it be:
```suggestion
        super().__init__(artifact_uri, tracking_uri=tracking_uri)
```

Reply:
Both `super().__init__(artifact_uri, tracking_uri)` and `super().__init__(artifact_uri, tracking_uri=tracking_uri)` work